config/locales/en.yml

```diff
@@ -1302,22 +1264,17 @@ en:
       button_text: "Pay now!"
       hints:
         donation: "Optional donation that will go only to the organization team"
-        payment_button: "Your payment information never reaches the WCA website. It is securely sent to Stripe, which handles the payment process and gets back to us."
```

Comment:
We should probably keep this(?)
Should be easy to add as a subtitle below the Stripe panel

Reply:
yes we can add that to the PaymentStep